app-engine/java/ultimate/src/com/google/cloud/tools/intellij/appengine/java/ultimate/server/run/AppEngineServerUltimateConfigurationType.java

```diff
@@ -36,10 +36,11 @@
 import org.jetbrains.annotations.Nullable;
 
 /** @author nik */
-public class AppEngineServerConfigurationType extends J2EEConfigurationType {
+public class AppEngineServerUltimateConfigurationType extends J2EEConfigurationType {
```

Comment:
Since we're renaming this, could we also consider adding `LocalServer` here if this is used solely as a local server configuration? This could make it a bit easier to understand this is local config straight from the class name.

Reply:
so your suggesting `AppEngineLocalServerUltimateConfigurationType` instead of `AppEngineServerUltimateConfigurationType`? 

I'm fine with that. I guess I should do it for the community one too then for consistency.

Reply:
yeah, something like that. I agree with the other suggestion, would be great for community one as well.

Reply:
done. good suggestion..that always bugged me too

Reply:
nice, thanks! definitely makes sense!